app/src/main/java/com/youssef/fixit/Auth/LoginFragment/LoginFragment.java

```diff
@@ -65,8 +65,6 @@ public void onClick(View view) {
     }
 
     private void initSharedPreferences() {
```

Comment:
No need for this method, try to clean up unused methods, For release builds progaurd will get rid of it but let's keep it clean and easy for progurd.